dspace-spring-rest/src/main/java/org/dspace/app/rest/converter/DiscoverFacetsConverter.java

```diff
@@ -50,61 +50,58 @@ public SearchResultsRest convert(Context context, String query, String dsoType,
         return searchResultsRest;
     }
 
-
-    private void addFacetValues(Context context, final DiscoverResult searchResult,
-                                final SearchResultsRest searchResultsRest,
-                                final DiscoveryConfiguration configuration) {
+    public void addFacetValues(Context context, final DiscoverResult searchResult, final SearchResultsRest resultsRest,
```

Comment:
As this method is now `public`, it'd be nice to add Javadocs to it.

Reply:
done